src/server/scripts/EasternKingdoms/ZulGurub/boss_mandokir.cpp

```diff
@@ -42,14 +36,18 @@ enum Says
 enum Spells
 {
     SPELL_CHARGE              = 24408, // seen
-    SPELL_OVERPOWER           = 24407, // Seen
-    SPELL_FEAR                = 29321,
-    SPELL_WHIRLWIND           = 13736, // Triggers 15589
-    SPELL_MORTAL_STRIKE       = 16856, // Seen
+    SPELL_OVERPOWER           = 24407, // seen
+    SPELL_FRIGHTENING_SHOUT   = 19134, // seen
+    SPELL_WHIRLWIND           = 13736, // triggers 15589
+    SPELL_MORTAL_STRIKE       = 16856, // seen
     SPELL_FRENZY              = 24318, // seen
     SPELL_WATCH               = 24314, // seen 24315, 24316
-    SPELL_WATCH_CHARGE        = 24315, // Triggers 24316
-    SPELL_LEVEL_UP            = 24312  //
+    SPELL_WATCH_CHARGE        = 24315, // triggers 24316
+    SPELL_LEVEL_UP            = 24312, // seen
+    SPELL_EXECUTE             = 7160, // seen
+    SPELL_MANDOKIR_CLEAVE     = 20691, // seen
+
+    SPELL_REVIVE              = 24341 // chained spirit - seen
```

Comment:
"seen" comments are pretty useless. Remove them

Reply:
@Nefertumm it was a chromiecraft's people think, so I just followed the standard, I also found them useless.
Resolved.

Reply:
Nope, it was not chromiecraft's.

Reply:
dumb thing resolved, let's move forward.